### PR TITLE
update favicon to gravatar, remove favivon file

### PR DESCRIPTION
**Pull request by jsegal205**

### CODE REVIEW

**Comment by camalot** [comment]
:-1: keep the :chicken: suit

File: index.html [hunk 1]
```diff
@@ -68,4 +68,5 @@
 			ga('send', 'pageview');
 		</script>
 	</body>
-</html>
\ No newline at end of file
+</html>
+`
```
Comment (camalot) [inline index.html]: you have tick mark after `</html>`

**Comment by camalot** [comment]
:+1:

**Comment by jsegal205** [comment]
this is so 🔥

**Comment by camalot** [comment]
:bomb:  :fire:  :fire_engine:

**Comment by jsegal205** [comment]
nye

**Pull request merged**